internal/controller/datadogagent/controller_v2_test.go

```diff
@@ -74,7 +74,7 @@ func TestReconcileDatadogAgentV2_Reconcile(t *testing.T) {
 		wantFunc func(c client.Client) error
 	}{
 		{
-			name: "DatadogAgent default, create Daemonset with core, trace and process agents",
+			name: "DatadogAgent default, create Daemonset with core and trace agents",
```

Comment:
Wasn't sure if I updated these tests in the intended way (same with `factory_test.go`. Please let me know 🙇

Reply:
Yeah, this looks good!